general_rules/Procedure.tex

```diff
@@ -72,15 +72,9 @@ \subsection{Collisions}
 	\item \textbf{\iterm{Major collisions}:} If a robot crushes into something during a test, the robot is immediately stopped.	Additional penalties may apply.
 
 	\item \textbf{\iterm{Functional touching}:} Robots are allowed to apply pressure on objects, push away furniture and, in general, interact with the environment using structural parts other than their manipulators.
-	This is known as \iaterm{functional touching}.
```

Comment:
Why is this change needed? `iaterm` is a valid command, see https://github.com/RoboCupAtHome/RuleBook/blob/5185332e20fa3fd4444a9930e4350ed2a1033290/setup/abbrevix.tex#L51

Reply:
Because with iaterm the result in pdf is "functional touching (.)"
As far as I can tell iaterm takes two arguments.